modules/react-components/src/layouts/list.tsx

```diff
@@ -280,7 +285,20 @@ export const ListLayout: FunctionComponent<PropsWithChildren<ListLayoutPropsInte
                 )
             }
             <div className="list-container">
-                { children }
+                { isLoading ?
+                    (
+                        <DataTable 
+                            isLoading={ true }
```

Comment:
Rather than creating a separate instance of DataTable for the loader - can't we reuse the same DataTable to render the children too? Will need to check this with the possible `children` elements too whether they are matching

Reply:
Ack. Will consider as a future improvement